src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -0,0 +1,92 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  background-color: var(--color-black);
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding-bottom: 150px;
+  padding-top: 100px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 38px;
+    padding-top: 50px;
+  }
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-column-start: 2;
+  grid-row: 2;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaBottom {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.title {
+  font: var(--font-title);
+  grid-column-end: 11;
+  grid-column-start: 2;
+  margin-bottom: 54px;
+}
+
+.statCard {
+  margin-right: 85px;
+}
+
+.number {
+  font: var(--font-family-lato);
+  font-size: 72px;
+  margin-bottom: 5px;
+  width: 250px;
+}
+
+.statement {
+  font: var(--font-body-medium);
+}
+
+@media (--tablet-and-down) {
+  .statement {
+    margin-bottom: 40px;
```

Comment:
Actually, I'd probably apply this to the `.statCard`, not the `.statement`, since it's really the stat cards that are being spaced apart. This will be visually identical, but I think it makes more sense. Compare:

<img width="458" alt="Screen Shot 2020-11-10 at 9 31 01 PM" src="https://user-images.githubusercontent.com/1002748/98772658-13439a80-239c-11eb-90a6-1390e8878920.png">

vs.

<img width="497" alt="Screen Shot 2020-11-10 at 9 31 14 PM" src="https://user-images.githubusercontent.com/1002748/98772664-18084e80-239c-11eb-8f31-808b087ea0ed.png">